web/src/pages/billing/BillingCostByMonth.tsx

```diff
@@ -173,7 +187,22 @@ const BillingCostByTime: React.FunctionComponent = () => {
         return CloudSpendCategory.COMPUTE_COST
     }
 
-    const getData = (query: BillingTotalCostQueryModel) => {
+    // Pre-fetch topics data on component mount
+    React.useEffect(() => {
+        const preFetchTopics = async () => {
```

Comment:
Similar comment in this to the category page component re using promise chaining rather than defining a function